components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -3481,6 +3481,17 @@ Bretherton; 'HBR' for Rasch modified version of 'HB'.
 Default: 'HB'
 </entry>
 
+<entry id="linearize_pbl_winds" type="logical" category="pbl"
+       group="phys_ctl_nl" valid_values="" >
+If .true., a linearization of the response of the pbl scheme to surface stresses
+is sent to the coupler every time step, using the fields 'wsresp' and 'tau_est'.
+</entry>
+
+<entry id="export_gustiness" type="logical" category="pbl"
+       group="phys_ctl_nl" valid_values="" >
+If .true., a gustiness value will be exported to the coupler.
+</entry>
```

Comment:
Seems like the description here could use more details about what this is for. I'm thinking just something like "helps stabilize near-surface wind oscillations" or something similar will help clarify why the option exists. It's a good idea to be verbose here for the sake of people who look at this in the future and have no idea what the various options are used for.

Reply:
Got it. I've added something a bit more verbose.